todo_app/app.py

```diff
@@ -1,11 +1,23 @@
-from flask import Flask
-
+from flask import Flask, render_template, redirect, request
+ 
 from todo_app.flask_config import Config
-
+from todo_app.data.session_items import get_items 
+from todo_app.data.session_items import add_item
```

Comment:
More a matter of preference than a strong recommendation, but worth noting that we can combine these imports if we wish like the Flask imports at the top:
```suggestion
from todo_app.data.session_items import get_items, add_item
```